packages/trader/src/AppV2/Components/TradeParameters/RiskManagement/take-profit-and-stop-loss-input.tsx

```diff
@@ -0,0 +1,328 @@
+                WS.forget(subscription?.id);
+                is_be_response_received_ref.current = true;
+                return;
+            }
+            if (subscription?.id) subscription_id_ref.current = subscription?.id;
+
+            const new_error = error?.details?.field === type ? error?.message : '';
+            setErrorText(new_error);
+            updateParentRef({
+                field_name: is_take_profit_input ? 'tp_error_text' : 'sl_error_text',
+                new_value: new_error,
+            });
+            if (error?.message) WS.forget(subscription?.id);
+
+            /* For Multipliers, validation parameters come in proposal response only if TP or SL are switched on and their value is not empty.
+            Here we set them into the state in order to show further even if we got a validation error from BE.*/
+            if (
+                isMounted() &&
+                proposal &&
+                echo_req.contract_type === CONTRACT_TYPES.MULTIPLIER.UP &&
+                (echo_req?.limit_order?.take_profit || echo_req?.limit_order?.stop_loss)
+            ) {
+                const { validation_params } = proposal as ExpandedProposal;
+                setInfo(info => {
+                    if (
+                        (info.min_value !== validation_params?.[type]?.min && validation_params?.[type]?.min) ||
+                        (info.max_value !== validation_params?.[type]?.max && validation_params?.[type]?.max)
+                    ) {
+                        return {
+                            min_value: validation_params?.[type]?.min,
+                            max_value: validation_params?.[type]?.max,
+                        };
+                    }
+                    return info;
+                });
+            }
+            is_be_response_received_ref.current = true;
+        };
+
+        /* In order to get validation params for Multipliers when TP and SL are empty, 
+            we send '1' first, get validation params and set them into the state.*/
+        const input_value = should_set_validation_params ? '1' : new_input_value;
+        const dispose = requestPreviewProposal(
+            trade_store,
+            onProposalResponse,
+            {
+                ...(is_take_profit_input ? { has_take_profit: is_enabled } : { has_stop_loss: is_enabled }),
+                ...(is_take_profit_input
+                    ? { take_profit: is_enabled ? input_value : '' }
+                    : { stop_loss: is_enabled ? input_value : '' }),
+            },
+            true
+        );
+
+        return () => {
+            dispose?.();
+            WS.forget(subscription_id_ref.current);
+            clearTimeout(focus_timeout.current);
+        };
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_enabled, new_input_value]);
+
+    // TODO: remove all regex replacement, when comma will be supported by quill component with type number
```

Comment:
I think we should wait for changes from quill side, we should not adjust the code this much

Reply:
After this Quill UI PR will be added to the latest lib version, I will update my PR
https://github.com/deriv-com/quill-ui/pull/391#pullrequestreview-2260022405

Reply:
Updated quill ui library to the latest version, removed extra logic with changing input value with reg exp